app/renderer/js/notification/helpers.js

```diff
@@ -83,6 +83,36 @@ function parseReply(reply) {
 		reply = reply.replace(regex, streamMention);
 	});
 
+	window.$.ajaxSetup({async: false});
+
+	window.$.getJSON('/json/users', data => {
+		const members = data.members;
+		const bots = [];
+		members.forEach(membersRow => {
+			if (membersRow.is_bot) {
+				botsList.push(membersRow);
```

Comment:
Right here instead of pushing `membersRow` here you should push `memberRow.full_name` and the mention syntax so you do not need to loop through it again.